tests/behat/behat-features/src/MediaReplicationContext.php

```diff
@@ -0,0 +1,168 @@
+   *
+   * @Then Unused media is not replicated
+   */
+  public function checkUnusedMedia() {
+    foreach (self::$createdMediaEntities as $id => $title) {
+      if (!in_array($id, self::$replicatedMediaEntities)) {
+        self::$unusedMedia = $id;
+        $found_element = $this->getSession()
+          ->evaluateScript("jQuery(\"*:contains('$title')\").length > 0");
+        if ($found_element) {
+          throw new ExpectationException('Element should not be found.', $this->getSession());
+        }
+        break;
+      }
+    }
+  }
+
+  /**
+   * @Given I click on :index of created media in :entity_browser
+   */
+  public function iClickOnIndexOfCreatedMediaInEntityBrowser($index, $entity_browser) {
+    $id = array_slice(array_keys(self::$createdMediaEntities), $index, 1)[0];
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$id\").length > 0");
+    if (!$found_element) {
+      throw new ExpectationException('Element not found.', $this->getSession());
+    }
+    $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$id\").closest(\".views-row\").click()");
+  }
+
+  /**
+   * @Given I click on edited media in :entity_browser
+   */
+  public function iClickOnCurrentOfCreatedMediaInEntityBrowser($entity_browser) {
+    $id = self::$currentlySelectedMedia;
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$id\").length > 0");
+    if (!$found_element) {
+      throw new ExpectationException('Element not found.', $this->getSession());
+    }
+    $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$id\").closest(\".views-row\").click()");
+  }
+
+  /**
+   * @Given I click on unused media in :entity_browser
+   */
+  public function iClickOnUnsedMediaInEntityBrowser($entity_browser) {
+    $unused = self::$unusedMedia;
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$unused\").length > 0");
+    if (!$found_element) {
+      throw new ExpectationException('Element not found.', $this->getSession());
+    }
+    $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$unused\").closest(\".views-row\").click()");
+  }
+
+  /**
+   * I edit last selected media.
+   *
+   * @Then I edit last media
+   */
+  public function editLastMedia() {
+    $id = self::$currentlySelectedMedia;
+    $xpath = "//td[contains(@class, 'field-operations')]//a[contains(@href, 'media/$id/edit')]";
```

Comment:
works with generic step definition "I click link below element selector". css can do attribute comparision and :contains - alternatively the step supports xpath also now.